Digipost/UIAlertController+APIError.swift

```diff
@@ -10,35 +10,22 @@ import UIKit
 
 extension UIAlertController {
     
-    class func presentAlertControllerWithAPIError(apiError: APIError, presentingViewController: UIViewController) {
+    class func presentAlertControllerWithAPIError(apiError: APIError, presentingViewController: UIViewController, didTapOkClosure: (() -> Void)? ) {
         if apiError.shouldBeShownToUser {
             let alertController = UIAlertController(title: apiError.alertTitle, message: apiError.altertMessage, preferredStyle: UIAlertControllerStyle.Alert)
             
             let okAction = UIAlertAction(title: "Ok", style: UIAlertActionStyle.Default, handler: { (alertAction) -> Void in
-                
+                didTapOkClosure?()
             })
-            
-            alertController.addAction(okAction)
-            presentingViewController.presentViewController(alertController, animated: true, completion: { () -> Void in
-                
-            })
-        }
-    }
-    
-    class func presentAlertControllerWithAPIError(apiError: APIError, presentingViewController: UIViewController, didTapOkClosure: () -> Void ) {
-        if apiError.shouldBeShownToUser {
-            let alertController = UIAlertController(title: apiError.alertTitle, message: apiError.altertMessage, preferredStyle: UIAlertControllerStyle.Alert)
-            
-            let okAction = UIAlertAction(title: "Ok", style: UIAlertActionStyle.Default, handler: { (alertAction) -> Void in
-                didTapOkClosure()
-            })
-            
+
             alertController.addAction(okAction)
-            presentingViewController.presentViewController(alertController, animated: true, completion: { () -> Void in
-                
-            })
-            
+
+            if presentingViewController.navigationController?.topViewController == presentingViewController && presentingViewController.presentedViewController == nil {
+                presentingViewController.presentViewController(alertController, animated: true, completion:nil)
+                var descriptionWithNoForbiddenCharacters = "\(presentingViewController)".stringByReplacingOccurrencesOfString("<", withString: "")
```

Comment:
removed logging which view that got the alert, i don't think the information is that valuable. It's easily understandable from the alert code also.
commit: 802fddbe390b5b38c97c6d731c00e182f4abbd35